_commands/basics/kill.md

```diff
@@ -3,22 +3,31 @@
 
 kill
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`kill` is used to send a signal to a process.
+
 ~~~ bash
-*short* example of command usage and output
+kill [-s] [-l] %pid
```

Comment:
Could you include a $ before the command, so it looks like `$ kill ...` instead of `kill`? And could you elaborate more on the use of the -s flag?